### PR TITLE
Add logic to generate beta version documentation

### PR DESCRIPTION
**Pull request by hui1110**
As title.

### CODE REVIEW

File: docs/src/main/asciidoc/ghpages.sh [hunk 1]
```diff
@@ -145,6 +154,17 @@ function copy_docs_for_current_version() {
             fi
         done
         COMMIT_CHANGES="yes"
+    elif [[ "${BETA_RELEASE_BRANCHES_VALUE}" == "feature/spring-cloud-azure-passwordless-connection" ]] ; then
```
Comment (saragluna) [inline docs/src/main/asciidoc/ghpages.sh:157]: here we should check wether current branch is one of the BETA_RELEASE_BRANCHES_VALUE

File: docs/src/main/asciidoc/ghpages.sh [hunk 2]
```diff
@@ -145,6 +154,17 @@ function copy_docs_for_current_version() {
...
+      echo -e "Current branch is '${BETA_RELEASE_BRANCHES_VALUE}' - will copy the current docs only to the root folder"
```
Comment (saragluna) [inline docs/src/main/asciidoc/ghpages.sh:158]: Current branch is `CURRENT_BRANCH`

Comment (saragluna) [reply docs/src/main/asciidoc/ghpages.sh:158]: The log info should be like, 

```
Branch [${CURRENT_BRANCH}] is a whitelisted beta release branch! Will copy the current docs to the root folder

```

**Review by saragluna — APPROVED** [review]

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**